spec/factories/users.rb

```diff
@@ -205,12 +205,30 @@
     end
 
     trait :with_pending_gpo_profile do
-      after :build do |user|
-        profile = create(:profile, :with_pii, gpo_verification_pending_at: 1.day.ago, user: user)
-        gpo_code = create(:gpo_confirmation_code)
-        profile.gpo_confirmation_codes << gpo_code
-        device = create(:device, user: user)
-        create(:event, user: user, device: device, event_type: :gpo_mail_sent)
+      transient do
+        code_sent_at { created_at }
+      end
```

Comment:
What does `transient` do? Create the variable temporarily and then not leave it as part of the returned object?

Reply:
Yep!